pkg/controller/authentication/authentication_controller.go

```diff
@@ -276,9 +283,13 @@ func (r *ReconcileAuthentication) Reconcile(ctx context.Context, request reconci
 	// Clear the degraded bit if we've reached this far.
 	r.status.ClearDegraded()
 
-	if !r.status.IsAvailable() {
-		// Schedule a kick to check again in the near future.
-		return reconcile.Result{RequeueAfter: 30 * time.Second}, nil
+	// TODO <before PR is closed> figure out a better way to handle the authentication resource not being "available"
```

Comment:
What's the status of this TODO?

Reply:
I think I know what I'm going to do, just need to confirm with Casey. I think we may just have another function for the status manager to say there's no resources to monitor.